libbeat/common/mapval/is_defs.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"time"
 
+	"strings"
```

Comment:
goimports fun

Reply:
Gah! I need to add a pre-commit hook. That's the only way I'll learn.